examples/nips17_adversarial_competition/validation_tool/README.md

```diff
@@ -0,0 +1,22 @@
+
+# Submission validation tool
+
+This tool could be used to verify whether submission file is valid or not.
```

Comment:
Maybe change to "This tool verifies that a submission file is valid or reports an error."